src/Schema/Directives/RulesDirective.php

```diff
@@ -28,6 +28,13 @@ public function getRules(): array
     {
         $rules = $this->directiveArgValue('apply');
 
+        // Resolve custom rule namespace, if possible.
+        foreach ($rules as $key => $rule) {
+            if (strpos($rule, 'App\Rules') === 0) {
```

Comment:
I think we might just use `class_exists()` to determine if we are dealing with a custom rule.